frontend/routes/viewerGui/components/issues/components/issueDetails/issueDetails.component.tsx

```diff
@@ -332,6 +361,79 @@ export class IssueDetails extends React.PureComponent<IProps, IState> {
 		}
 	}
 
+	public handleUpdateScreenshot =
```

Comment:
@sanmont3drepo @carmenfan I agree and I am also aware that both of these panels are largely duplicated, but have minor differences and inconsistencies, therefore I think that writing it in a more DRY way would require a larger refactor and would be more time-consuming than it seems to be at first glance. 
However, if you think that it would be worth taking care of it on this PR, let me know.

Reply:
mm ok, lets tackle the duplication problem in a refactor branch